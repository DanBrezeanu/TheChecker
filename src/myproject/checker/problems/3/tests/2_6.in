203728056 246735616 103734407 538555249 732354292 75784424 906649190 118868130 716094529 478546472 483737096 187468480 703973451 659956655 70642404 493379297 395988703 990252578 127784001 136520457 317957393 863061598 840806928 726178636 976929660 92972407 984584905 105854953 31174620 679030915 889101450 63920899 797868652 719360297 171145891 173442537 441554395 677156680 976904465 395337593 591669702 903263204 572276396 860900968 377191528 471146808 232236332 4510689 727821615 976219320 864128540 743785089 369059345 450206369 236377195 889249925 402664924 459079552 923316735 243147548 453384875 64338283 19066360 175632428 319173542 73216203 695402305 432348486 131531914 398438485 46722992 873695646 445123392 130843971 586698080 712503786 135926519 594455573 89657090 411038522 897399702 330646351 507340953 77372041 200061999 925733205 33345331 578780878 65729428 999903412 893124942 208727498 208293010 450075010 128430066 533998862 46919414 784528601 39480106 641028626 706033973 559305605 662359407 211988042 883547133 570845986 399794490 956034061 703185724 934345657 173758339 462726567 493539832 814857409 211348949 832161565 381431016 177735148 146849286 20894661 446373262 604790008 84871451 163849510 527785932 796223824 798418417 591051677 614204047 387853515 203756193 108537351 271031377 916349986 778669185 70431730 473862037 417247854 613919850 777461463 201902978 319695418 534803638 673878102 799246825 243226043 281318463 926597650 528272243 35483026 344642799 375387381 49188295 992496426 159138315 997473658 670392900 982611567 68913717 31224528 552582989 249885769 494298137 125315254 418733339 146498225 226563327 983848388 848048417 423268807 708343870 103749310 59273907 149428375 728555036 640515901 582532885 324309466 24529423 879071904 857638540 993358225 848883471 203527396 220950148 701195162 681294915 256630981 459386153 595473804 245434105 168159308 942490240 565589206 166724601 831243284 849959564 135486260 635381125 614039902 332538811 41733777 334837638 639447404 346909836 687363224 761671612 686192949 812236910 866616328 807022174 937744255 944313319 315690251 852146861 905045921 27560290 194714527 559695279 278443396 613124088 845952091 569507654 877497709 682132076 837559201 512345654 387342396 412459260 860915675 622526373 503223946 442512795 927015827 629745376 155617253 472861242 23516456 869721228 841895128 582993310 417793732 873134915 742465886 654305046 268814778 583801600 645836008 334810387 214752460 752339088 471287433 633064554 668349859 72777573 339623612 84677422 29036775 923158478 518032940 294266081 725452322 601712100 893273313 2564628 262303107 446077621 449580391 115600953 496768024 784192674 182598112 852975459 394979625 987899993 532508492 326673018 136107210 389473248 419817686 447847329 953596211 685609521 520251380 714355327 925563368 254946029 889988384 86812774 232267797 574735016 349080691 75514363 932656057 434353673 564423956 959379959 623859953 340656100 882152771 611413393 134201175 949879 550556153 999459281 860934477 459450426 311883965 613152115 676227934 705055916 221754871 584459884 459196543 9087463 768619801 622710163 114017573 684774365 562504017 980490787 769811183 576356365 640903493 525699330 457340531 401346236 144557896 886509339 173334256 294349346 904130013 179862421 357956378 640452958 945915148 361712130 94263958 144395365 133173482 982663332 901425860 66722350 795514960 473298043 79209608 787789436 237941645 285717621 436497707 664915384 707347309 201480269 517060546 986251301 57396284 273438907 721994467 758419664 805527816 658329017 513724559 527176570 587786329 625254963 192402270 832487783 449064787 350091567 157420748 855005198 118485134 412382883 473108982 537999188 300651799 935833053 524019290 391792689 807751026 53019301 624013249 384611155 108457081 901079081 916902814 451282963 143808018 592298304 196708751 827353298 582412969 263570888 952277748 568271468 403737314 721159221 226253202 482654657 767491548 682690663 295235877 561291641 270107868 993551709 578119359 760825599 558720069 143060910 580378476 745402700 772190945 378349610 348534797 532429176 507418341 114989173 402790378 669410828 377437379 534808505 566276402 672992216 103854883 357016596 168105972 416398872 564290911 377315661 559924871 774698111 766006267 575910402 973145589 347935491 736576411 955554665 974400476 526470920 290718473 178033341 555606249 184647177 823319951 159980138 572172511 409041829 583386736 878132966 327280196 947818356 123839594 2298071 317943527 132582797 418426459 712610383 738609789 909604018 204656756 675300152 925924418 780183006 392319665 185716463 746264676 926017610 505946703 40324789 153444751 201096599 308298838 271365327 121559738 771915503 430258161 507672586 672044130 734431950 590709020 494297218 110441938 492260544 387946725 614533995 981739862 123491089 57542118 344548677 648462915 704131518 115975609 756235033 632813652 702140844 269198319 694136704 992832981 605710998 936585517 646723191 422130437 94126315 978902249 73853604 116579626 48455846 809284515 933966624 214502767 7364037 544466855 878983255 310430117 222072933 20617578 227462929 391044481 712951862 825017245 285514474 576236103 82062596 7972177 959421811 750291175 167591899 142979632 125732175 823989611 357251800 221656524 625747951 379335024 813641092 331141702 289313337 696151296 560742842 78190158 65936192 368848777 252917279 409383448 790282924 260794060 203119319 184747596 196219150 314005628 920392194 986519611 196802981 891625432 803525267 193694866 435500142 278535623 77747207 504656571 17770541 988877778 366546949 435252993 916866693 308515448 722864941 373354641 555633250 94247876 109060072 96701244 964961529 424795326 269214156 712700172 825590611 889430208 96269419 885115754 837589799 361749885 261908814 907255268 749944165 533934317 708313626 182224792 449403750 218554268 518134375 355135755 169775106 722300787 737080243 40891976 776741906 215969278 868006765 460159323 968328742 385846153 44044435 474711012 90353215 339979056 150435120 462611961 147593783 244739902 762415552 740416992 403304528 166902571 304058062 882378072 543937685 998916163 325544094 591303667 106526968 611910498 946241470 440491292 742136643 275763644 927985694 27315853 720172087 433178976 623196528 188494210 357455102 855257159 16407231 725533713 872141208 218212481 420807251 492441126 528965781 292797076 762306563 584121164 365812252 879267243 210976143 697719954 725268005 929901169 583830590 47845643 544416524 783736540 860838097 176523122 519480208 307678116 834519552 759503715 376908016 868007425 844993053 27468954 353606672 185675790 527672356 1969963 204773497 952332444 795211064 715956026 704276237 480788081 125986847 124217965 964469845 198257219 271196297 362448394 865192548 66154026 112986760 481957634 714573435 260201295 567100032 955947198 633526855 175804887 47967938 148245153 711641822 120685277 441990987 307769958 807018898 5875515 771713972 702658780 165920963 56389633 571228501 397754434 428742491 412838654 217229868 892107463 655115786 698111101 469571538 295567094 245550939 716221352 396069697 343717710 436729348 673724258 455417933 331318117 213432243 820612178 409141091 543474886 393065347 341327222 367982347 662274160 668088012 312740406 145270339 750117332 693482217 480687616 46399535 873790009 919466634 136167718 424528113 829004095 974970248 43876798 981784252 446282867 67986433 292489509 202579924 583397513 104226717 871004459 166260865 562308179 162963797 81421284 458634242 482904413 72327543 72766187 372320387 745573073 286010084 321542301 827295215 161797276 374222412 199350944 444966740 692240597 54395659 902708747 719191009 501890586 115016456 378364520 51979037 662789241 609631243 523159412 500245689 843866872 823759985 887034344 57667060 152786871 614873175 437970130 524352700 743023626 935009726 661637364 452310974 527850530 743199847 396488093 380237497 485010827 359072273 920977596 940996295 913039687 625103900 243719565 364626664 736618132 972746654 579854156 861332699 330753292 890804798 814699961 915655907 329779968 524535020 882075118 236867439 845694335 840364084 944085349 174378387 911753023 532641286 485287159 444640573 416175613 530140200 522915638 367411418 412747593 387465013 732268293 470617122 722906001 281753829 922723729 254637939 383304484 925715630 414161103 456955959 21482161 862851137 426898669 184576866 907953937 658867511 659049582 587030956 393683558 413213598 381608295 754873438 491833353 474268315 602196431 330429571 119587869 684358045 610887126 229249537 885987781 621283625 872500382 888919243 502345233 340109921 642777414 994295099 51114864 86221851 504572291 902261614 454221509 974703970 582946473 821436755 852935111 399948954 722645300 696537231 340219769 994167548 847922616 559378526 52737476 886985303 357256213 102252403 450432708 970039618 86785224 541553151 646624896 230957205 144533843 540160527 250745256 441527617 88239947 618582892 310027810 563113074 765202183 884927186 387956130 641750642 591974166 22613753 562106752 529046388 258354027 536304945 74877756 421983245 581846423 904467213 395199959 204884055 547457473 137267100 218492293 629769566 113430083 970212523 587074044 273053759 786957854 581288002 49795311 308692488 27000544 810976860 459963280 295899989 869983305 814756082 798197504 209538644 578524662 910919144 282606724 338455763 907000489 129816515 441343070 825574631 660800536 728700241 708156296 414667474 618132621 444531724 236943053 42653733 328539706 613810718 752398683 133692014 372609975 505193928 268782103 256854548 890343108 999721626 488014583 549361795 160758813 141326374 616626791 644271156 884975778 489520830 266298579 449741423 300289452 328518512 318340108 52787872 971713258 96024013 79479187 713325490 233200648 214702242 349592121 528892693 445633777 181050696 564449731 529757296 609935902 731995431 400140491 728970071 591893594 785392484 996723655 595876954 885307032 334761544 139143591 397462067 386380572 761521615 212481248 294900309 171497996 323822466 219547756 209239288 164572247 952655102 889209892 131351981 765696289 123051993 985418873 603467726 944062055 539856753 245501527 457102517 359292677 48937569 59877504 102110106 699547734 465412591 500942551 806369718 193679424 547475660 699666402 145282961 806501061 572433870 551432202 648830849 380259368 133743296 130853905 702310295 825050762 278432589 192047946 34026465 174042920 149492013 298857 42698909 503980735 677631966 341675678 179625897 860848495 613356673 750433136 632870517 140106084 393181234 877169098 91492433 995964151 415958303 397802336 837343855 561293943 419462598 251221488 228190710 563221244 863488051 268418517 82413140 97704240 532170827 849437090 303290373 429571356 927040633 334832479 977984459 336943164 937702394 172104828 614691806 670998494 203007371 556854146 504985691 938553413 143377670 270462532 689451818 276007291 421843253 332359710 863306118 942143127 107524602 898698400 712726258 629449986 959411589 34144729 87739055 40887612 285361033 782195617 329137425 572172654 105895859 128330237 522535597 226980071 305210547 690254333 60118722 30343226 288927290 790101472 7564217 598194125 880485505 948409388 876627774 776086972 186466058 538679399 314650358 651650637 268505408 714777111 247791494 698303478 523556034 738956081 641849828 700044345 258571550 846011738 62855647 264942427 354379053 82893600 544835799 970353813 43980575 9097070 233079444 869825861 856852891 578046695 367706594 380153499 66745295 16146138 231474421 573389427 635490585 192149165 625802673 543161873 793483568 421733785 939762134 22156440 331756544 311260946 181591812 903332286 240675513 464883866 412352009 584101271 279623557 291169011 975756333 900415517 955294538 416709521 459512933 13360505 84454928 726410843 840239218 286372812 500417770 61666964 185024872 418077088 406767464 274119993 897402507 257799115 766290140 267197191 155502430 561366470 973574777 954208311 919938041 776366153 930565812 65639154 110964906 802054227 390515192 889112172 993105515 562312560 41107599 754874814 973973564 712960293 311008612 901753095 755616600 303762765 48485602 911330428 414934299 375693024 940458831 390667439 138711857 811268485 819153655 283332018 321856221 822118122 361534385 399589107 127891868 176737876 715723705 231092745 610181170 739993682 702451099 133227463 552067676 839404362 126397566 650147818 903873458 471963007 38056694 703168409 246288352 69045599 357057144 81941949 452271344 947911954 378107801 730616461 784417649 101336165 693416293 836542773 921600623 959240639 986874570 200627101 969994348 756263458 181004630 976673949 922569599 982638866 709944078 297717043 703356690 160187689 770426991 486797746 702939279 611573432 321449458 48672358 704876017 258258166 825065882 616478422 324067528 268547780 772893328 832701076 848988958 262901697 145650458 172054162 378082888 946859251 526747763 226988664 95726994 711667435 148807772 5533693 230930999 656908080 90556538 946453930 327122042 230146137 277240227 654815739 810605324 0
