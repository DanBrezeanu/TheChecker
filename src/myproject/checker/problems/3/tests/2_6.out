498049614.22
